src/Support/SelectFields.php

```diff
@@ -28,7 +32,10 @@ class SelectFields
     private $select = [];
     /** @var array */
     private $relations = [];
-
+    /**
+     * @var array<class-string<Model>,array<string,string>>
+     */
+    private static $modelRelationshipsForAlias = [];
```

Comment:
Wish this wouldn't be static.

I understand from the code it's cleaned up, still…

Reply:
yeah me too